primitives/state-machine/src/basic.rs

```diff
@@ -111,6 +111,10 @@ impl From<BTreeMap<StorageKey, StorageValue>> for BasicExternalities {
 }
 
 impl Externalities for BasicExternalities {
+	fn local_ocw_storage_write_kv(&mut self, _key: &[u8], _value: &[u8]) {
+		warn!("basic externalities does not support local_ocw_storage_write_kv")
```

Comment:
Under what circumstances is this `warn!` triggered? Is this codepath hit whenever the `offchain_indexing` flag is false and we import a block whose runtime calls this function?

Reply:
I removed the warning, should not have been there in the first place.